app/views/interview/index.html.erb

```diff
@@ -5,6 +6,28 @@
 		<td>Interviewer Name</td>
 		<td>Datetime</td>
 		<td>Location</td>
+		<td>Cancel</td>
+	</tr>
+	<% @my_interviews.each do |i| %>
+	<tr>
+		<td><%= link_to i.id, interview_path(i) %></td>
+		<td><%= i.applicant_name %></td>
+		<td><%= i.interviewer_name %></td>
+		<td><%= i.time %></td>
+		<td><%= i.location %></td>
+		<td><%= link_to "Cancel", interview_cancel_path(i.id) %></td>
+	</tr>
+	<% end %>
+</table>
+<h2>All Interviews</h2>
```

Comment:
Yeah that's a good idea. I'll move it to /interview/admin.

I put everything together originally to eliminate duplicate views since they all look the same.